src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 31px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__img {
+    width: 160px;
+    height: 135px;
+    align-self: center;
+    background-image: url(../images/imac.jpeg);
+    background-size: 160px 134px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-bottom: 40px;
+  }
+
+  &__details {
+    margin-bottom: 16px;
+  }
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 0;
+    margin-bottom: 4px;
+    color: $accentColor;
+    line-height: 18px;
+  }
+  &__description {
+    font-weight: 400;
+    font-size: 10px;
+    color: $secondColor;
+    margin: 0;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+    &-text {
+      display: inline-block;
+      margin: 0;
+      color: $accentColor;
+      line-height: 16px;
+      align-self: baseline;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    line-height: 18px;
+
+    &-title {
+      font-size: 12px;
+      margin: 0;
+
+      color: $secondColor;
+    }
+    &-price {
+      font-size: 16px;
+      font-weight: 700;
+      margin: 0;
+      color: $accentColor;
+    }
+  }
+
+  &__link {
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    color: #fff;
+    text-align: center;
+    text-decoration: none;
+    line-height: 40px;
+
+    display: block;
+    height: 40px;
+    box-sizing: border-box;
+    width: 100%;
+    background-color: #00acdc;
```

Comment:
Create a variable for repeated color and use it everywhere

Reply:
Not fixed